File: expressions/financial_calculator.py
# Paige Jolley, Financial Calculator Python

# print stament that welcomes user and tells what the program does
print("Welcome! This is a Financial Calculator.")

# ask what their income is (variable and input)
income = float(input("What is your income?\n"))

# ask what their rent is (variable and input)
rent = float(input("What is your rent? \n"))

# ask what their utilities is (variable and input)
utilities = float(input("What is your utility bills? \n"))

# ask what their groceries is (variable and input)
groceries = float(input("What is the cost of your groceries? \n"))

# ask what their trasportation is (variable and input)
transportation = float(input("What is your trasportation cost? \n"))

# calculate savings as 10% of income (income*.1) (variable)
savings = income*.1 

# calculate spending as income-savins-rent-untilites-groceries-transportation (variable)
spending = income-savings-rent-utilities-groceries-transportation

# calculate percent income of rent (rent/income *100) (variable)
percent_income_of_rent = rent/income *100

# calculate percent income of utilites (utilities/income *100) (variable)
percent_income_of_utilites = utilities/income *100

# calculate percent income of groceries (groceries/income *100) (variable)
percent_income_of_groceries = groceries/income *100

# calculate percent income of transporation (transportation/income *100) (variable)
percent_income_of_transportation = transportation/income *100

# calculate percent income of spending (spending/income *100) (variable)
percent_income_of_spending = spending/income *100

# Your rent is $XX.XX and which is XX% of your income. (print)
print(f"Your rent is {rent} an which is {percent_income_of_rent} % of your income. \n")

# Your utilities is $XX.XX and which is XX% of your income. (print)
print(f"Your utilities is {utilities} and which is {percent_income_of_utilites} % of your income. \n")

# Your groceries is $XX.XX and which is XX% of your income. (print)
print(f"Your groceries is {groceries} and which is {percent_income_of_groceries} % of your income. \n")

# Your transportation is $XX.XX and which is XX% of your income. (print)
print(f"Your transportation is {transportation} and which is {percent_income_of_transportation} % of your income. \n")

# Your savings is $XX.XX and which is XX% of your income. (print)
print(f"Your savings is {savings} and which is {savings} % of your income. \n")

# Your spending is $XX.XX and which is XX% of your income. (print)
print(f"Your spendings is {spending} and which is {percent_income_of_spending}% of your income. \n")

File: expressions/financial_calculator.c
// Paige Jolley, Financial Calculator C
#include <stdio.h>
#include <math.h>

int main(void){
    printf("Welcome! This program will be your financiL calculator.\n");
    int income;
    printf("What is your income?\n");
    scanf("%d", &income);

    int rent;
    printf("What is your rent?\n");
    scanf("%d", &rent);

    int utilities;
    printf("What is the cost of your utilities?\n");
    scanf("%d", &utilities);

    int groceries;
    printf("What is the cost of your groceries?\n");
    scanf("%d", &groceries);

    int transportation;
    printf("What is the cost of your transportation?\n");
    scanf("%d", &transportation);

    float savings = income*.1;

    float spending = (float)income-savings-rent-utilities-groceries-transportation;

    float per_rent = (float)rent/income *100;

    float per_utilities = (float)utilities/income *100;

    float per_groceries = (float)groceries/income *100;

    float per_transportation = (float)transportation/income *100; 

    float per_spendings = (float)spending/income *100;

    printf("Your rent is $%d and which is %.2f%% of your income.\n", rent, per_rent);
    
    printf("Your utilities is $%d and which is %.2f%% of your income.\n", utilities, per_utilities);

    printf("Your groceries is $%d and which is %.2f%% of your income.\n", groceries, per_groceries);

    printf("Your transportation is $%d and which is %.2f%% of your income.\n", transportation, per_transportation);

    printf("Your savings is $%.2f and which is 10%% of your income.\n", savings);

    printf("Your spending is $%.2f and which is %.2f%% of your income.\n", spending, per_spendings);
    

}
